src/metpy/__init__.py

```diff
@@ -4,21 +4,32 @@
 """Tools for reading, calculating, and plotting with weather data."""
 
 # What do we want to pull into the top-level namespace?
-
-import logging
+import sys
 import warnings
 
+if sys.version_info < (3,):
```

Comment:
This is going to bug me, but it's part of the recommendations and we should probably do it. Annoying because this code will be flagged as having no coverage.

Reply:
Agreed.